component/client-handler/src/main/java/org/wso2/carbon/identity/oauth2/token/handler/clientauth/jwt/validator/JWTValidator.java

```diff
@@ -109,10 +109,24 @@ public JWTValidator(boolean preventTokenReuse, String validAudience, int rejectB
      *
      * @param signedJWT Validate the token
      * @return true if the jwt is valid.
-     * @throws IdentityOAuth2Exception
+     * @throws OAuthClientAuthnException
      */
+    @Deprecated
```

Comment:
in the java doc let's point out what is the new method which needs to be used.

Reply:
Fixed with https://github.com/wso2-extensions/identity-oauth-addons/pull/88/commits/2124689721ed04d73e1e579e5b12ee117d6b352c